config/settings.yml

```diff
@@ -78,6 +78,11 @@
   :use_vim_broker: true
 :container_scanning:
   :concurrent_per_ems: 3
+  :by_provider_name:
+    :_default:
```

Comment:
> WRT this strategy to store per-provider settings I don't find it particularly interesting for several reasons:
> 
> it's hard to maintain
> the association by name doesn't seems reliable enough
> we're moving in a direction where we would like all these settings to be configurable with rest-api calls
> 

@simon3z For this patch this is the only solution I could think of so far. I think that with the default option we have a fallback and users will have to maintain their settings for specific providers. This is still WIP currently anyway until we will find a satisfactory solution to this problem.